design/apidsl/attribute.go

```diff
@@ -235,6 +238,7 @@ func parseAttributeArgs(baseAttr *design.AttributeDefinition, args ...interface{
 	return dataType, description, dsl
 }
 
+// Header can be used in: Headers, APIKeySecurity, JWTSecurity
```

Comment:
Should document `Member` below, can appear in `Payload`, `Param` can appear in `Params`, `Default` can appear in `Attribute`